src/wp-includes/feed-atom-comments.php

```diff
@@ -27,20 +27,30 @@
 		do_action( 'atom_comments_ns' );
 	?>
 >
-	<title type="text">
 	<?php
+	$feed_title = '';
 	if ( is_singular() ) {
 		/* translators: Comments feed title. %s: Post title. */
-		printf( ent2ncr( __( 'Comments on %s' ) ), get_the_title_rss() );
+		$feed_title = sprintf( ent2ncr( __( 'Comments on: %s' ) ), get_the_title_rss() );
 	} elseif ( is_search() ) {
 		/* translators: Comments feed title. 1: Site title, 2: Search query. */
-		printf( ent2ncr( __( 'Comments for %1$s searching on %2$s' ) ), get_bloginfo_rss( 'name' ), get_search_query() );
+		$feed_title = sprintf( ent2ncr( __( 'Comments for %1$s searching on %2$s' ) ), get_bloginfo_rss( 'name' ), get_search_query() );
 	} else {
 		/* translators: Comments feed title. %s: Site title. */
-		printf( ent2ncr( __( 'Comments for %s' ) ), get_wp_title_rss() );
+		$feed_title = sprintf( ent2ncr( __( 'Comments for %s' ) ), get_bloginfo_rss( 'name' ) . get_wp_title_rss() );
```

Comment:
This changes the substitution from `get_wp_title_rss` to `get_bloginfo_rss( 'name' ) . get_wp_title_rss()` which doesn't seem related to the ticket's purpose.